app/scripts/modules/ecs/src/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html

```diff
@@ -78,6 +93,57 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="sm-label-left">
+      <b>Placement Constraints</b>
+      <help-field key="ecs.placementConstraints"></help-field>
+    </div>
+    <form name="ecsPlacementConstraints">
+      <table class="table table-condensed packed tags">
+        <thead>
+          <tr>
+            <th style="width: 25%">Type <help-field key="ecs.placementConstraintType"></help-field></th>
+            <th style="width: 68%">Expression <help-field key="ecs.placementConstraintExpression"></help-field></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr ng-repeat="constraint in $ctrl.command.placementConstraints">
+            <td>
+              <ui-select ng-model="constraint.type" class="form-control input-sm">
+                <ui-select-match>{{ $select.selected }}</ui-select-match>
+                <ui-select-choices repeat="item in ['distinctInstance', 'memberOf'] | filter: $select.search">
+                  <span ng-bind-html="item"></span>
+                </ui-select-choices>
+              </ui-select>
```

Comment:
Since this is just a select with two options (both strings), it might be easier to just do:
```html
  <select 
    class="form-control input-sm" 
    ng-model="constraint.type" 
    ng-options="type for type in ['distinctInstance', 'memberOf']
  />
```
rather than do the whole `ui-select` thing

Reply:
Updated, much simpler!